### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by keshavvishwkarma**
Added KVRootBaseSideMenu-Swift

### CODE REVIEW

**Comment by keshavvishwkarma** [comment]
I have updated package.json for [ **issue**](https://github.com/IonicaBizau/made-in-india/issues/6)"
Please merge it.

File: package.json [hunk 1]
```diff
@@ -5808,10 +5808,10 @@
             "🌍"
           ],
           "rows": [
-            [
-              "5",
```
Comment (IonicaBizau) [inline package.json]: Can you please still keep this one?

**Comment by keshavvishwkarma** [comment]
I have kept that one.

**Comment by IonicaBizau** [comment]
Great! Going to merge it now!

**Comment by keshavvishwkarma** [comment]
Yeah! Thank you.

**Pull request merged**